src/project/flowr-analyzer.ts

```diff
@@ -147,12 +144,38 @@ export class FlowrAnalyzer<Parser extends KnownParser = KnownParser> implements
 		this.cache.reset();
 	}
 
-	public addRequests(requests: readonly RAnalysisRequest[]): void {
-		this.ctx.addRequests(requests);
+	public add(request: RAnalysisRequest | readonly RAnalysisRequest[] | `${typeof fileProtocol}${string}` | string): this {
```

Comment:
FYI: WebStorm now shows this a as duplicate code, but I did not see a benefit in sharing code between the analyzer and the builder